README.md

```diff
@@ -1,3 +1,23 @@
+# Zaken frontend
+
+## Production
+[zaken.amsterdam.nl](https://zaken.amsterdam.nl)
+
+## Acceptance
+[acc.zaken.amsterdam.nl](https://acc.zaken.amsterdam.nl)
+
+## Development
+- there is a dependency on https://github.com/Amsterdam/zaken-backend
+
+- `git clone https://github.com/Amsterdam/zaken-frontend.git`
+- `cd zaken-frontend`
+- `npm install .`
+- `npm run generate:swagger-schema`
+- `npm run generate:form-scaffolds`
+- `npm run start`
```

Comment:
Nice!

Do we want to mention the dependency on TOP as well?

Reply:
I would say Top (backend) is a dependency for zaken-backend. Both `npm run generate:swagger-schema`
and `npm run generate:form-scaffolds` only require zaken-backend, right? That's why I listed the dependency in the first place.